server/pulp/plugins/util/publish_step.py

```diff
@@ -577,9 +602,11 @@ def get_distributor_type(self):
 
     def publish(self):
         """
+        DEPRECATED
```

Comment:
I'd love to remove it, but I'm sure there are plugins that are calling it. I think it'll be easier to deprecate a bunch of methods, then remove them later (like in 3.0).